scripts/init-testnet.ts

```diff
@@ -484,15 +487,15 @@ function constructForeignAssetSetup(api: ApiPromise) {
                 symbol: "LKSM",
                 existentialDeposit: 0,
                 location: {
-                    V1: {
+                    V3: {
                         parents: 1,
                         interior: {
                             X2: [
                                 {
                                     Parachain: 2000
                                 },
                                 {
-                                    GeneralKey: "0x0083"
+                                    GeneralKey: "0x008300000000000000000000000000000000000000000000000000000000000000"
```

Comment:
I'm not sure about this - it seems to me the length is incorrect. But if I remove two zeroes, I get
```
 1:: Enum(GeneralKey):: decodeU8aStruct: failed at 0x83000000000000000000000000000000… on data (index 2/2): [u8;32]:: Expected input with 32 bytes (256 bits), found 31 bytes
```